app/src/main/res/values/strings.xml

```diff
@@ -169,160 +169,163 @@
     <string name="autofill">Autofill</string>
     <string name="autofill_service_name">KeePass DX Autofill Service</string>
     <string name="autofill_sign_in_prompt">Sign in with KeePass DX</string>
     <string name="set_autofill_service_title">Set default Autofill service</string>
-    <string name="set_autofill_service_summary">Enable the service to easily fill out forms from other applications</string>
+    <string name="set_autofill_service_summary">Enable the service to easily fill out forms from other apps</string>
     <string name="password_size_title">Password size</string>
     <string name="password_size_summary">Set the default size of the generated password</string>
     <string name="list_password_generator_options_title">Password characters</string>
     <string name="list_password_generator_options_summary">Set the default password generator characters</string>
     <string name="notifications">Notifications</string>
     <string name="clipboard_notifications_title">Clipboard Notifications</string>
     <string name="clipboard_notifications_summary">Enable clipboard notifications to copy entry fields</string>
-    <string name="clipboard_warning">Some devices are not able to automatically delete clips from the clipboard. If your manager does not allow this deletion, you will have to manually delete the copied item from your clipboard history.</string>
+    <string name="clipboard_warning">If your device is not able to delete clips from the clipboard automatically, as some are, delete the copied item from your clipboard history manually.</string>
     <string name="lock">Lock</string>
     <string name="lock_database_screen_off_title">Screen lock</string>
     <string name="lock_database_screen_off_summary">Lock the database when the screen is off</string>
-    <string name="fingerprint_quick_unlock_title">How to configure fingerprint for quick unlocking ?</string>
+    <string name="fingerprint_quick_unlock_title">How to configure fingerprint for quick unlocking?</string>
     <string name="fingerprint_setting_text">Set your personnal fingerprint for your device in </string>
-    <string name="fingerprint_setting_way_text">Settings -&gt; Security -&gt; Fingerprint</string>
+    <!-- Use ← for LTR languages -->
+    <string name="fingerprint_setting_way_text">\"Settings\" → \"Security\" → \"Fingerprint\"</string>
     <string name="fingerprint_type_password_text">Type your password in Keepass DX</string>
     <string name="fingerprint_scan_to_store">Scan your fingerprint to store your master password securely</string>
     <string name="fingerprint_scan_to_open">Scan your fingerprint when the password checkbox is unchecked to open the database</string>
     <string name="usage">Usage</string>
     <string name="fingerprint">Fingerprint</string>
-    <string name="fingerprint_enable_title">Fingerprint listening</string>
+    <string name="fingerprint_enable_title">Scanning for fingerprints</string>
```

Comment:
Not 100% sure about this. Seems the title suggests something other than what the content is…

Reply:
This is the activation title of the fingerprint reader in the settings.

Reply:
I think scanning works well for that.